src/js/media-seek-backward-button.js

```diff
@@ -1,6 +1,7 @@
 import MediaChromeButton from './media-chrome-button.js';
 import { defineCustomElement } from './utils/defineCustomElement.js';
 import { mediaUIEvents } from './media-chrome-html-element.js';
+import { Document as document } from './utils/server-safe-globals.js';
```

Comment:
It's now pretty unlikely you'll every use an element without media-controller, so I wonder if we just move this shim there more globally. (and obviously I keep forgetting)

Not needed for this PR though.